src/components/BookDB.jsx

```diff
@@ -0,0 +1,22 @@
+const primaryBooks = [
+  {
+    item_id: 'item1',
+    title: 'The Great Gatsby',
+    author: 'John Smith',
+    category: 'Fiction',
+  },
+  {
+    item_id: 'item2',
+    title: 'Anna Karenina',
+    author: 'Leo Tolstoy',
+    category: 'Fiction',
+  },
+  {
+    item_id: 'item3',
+    title: 'The Selfish Gene',
+    author: 'Richard Dawkins',
+    category: 'Nonfiction',
+  },
+];
+
+export default primaryBooks;
```

Comment:
### OPTIONAL 

- [ ] It will better to delete the array of books since books are currently fetched from the API 👍🏿